Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.3-onbuild
+
+EXPOSE 9102
+EXPOSE 9125/udp
+
+VOLUME ["/mapping.conf"]
```

Comment:
Actually this isn't even working since Volume definition in Dockerfile creates a directory, so if you don't bind mount you can't even place a file there:

```
FROM ubuntu
VOLUME [ "/foo.txt" ]
```

Build and run:
```
$ docker run -ti test bash
$ docker run test sh -c 'echo foo > /foo.txt'
sh: 1: cannot create /foo.txt: Is a directory
```